apps/src/applab/Visualization.jsx

```diff
@@ -20,6 +20,16 @@ var styles = {
   },
   phoneFrame: {
     marginBottom: 0
+  },
+  screenBlock: {
+    backgroundColor: 'rgba(0, 0, 0, 0.05)',
```

Comment:
Re: iterating style, `rgba(255, 255, 255, 0.05)` might give you a nicer 'fade' effect.